draft-js-emoji-plugin/src/components/EmojiSelect/Popover/Groups/Group/index.js

```diff
@@ -14,9 +14,26 @@ export default class Group extends Component {
     onEmojiSelect: PropTypes.func.isRequired,
     onEmojiMouseDown: PropTypes.func.isRequired,
     useNativeArt: PropTypes.bool,
+    isActive: PropTypes.bool,
   };
 
-  shouldComponentUpdate = () => false;
+  state = {
+    hasRenderedEmoji: false,
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isActive) {
+      this.setState({ hasRenderedEmoji: true });
+    }
```

Comment:
`willReceiveProps` is marked as `UNSAFE` in react 16 and will be deprecated

@vybu would you mind updating this to use [`getDerivedStateFromProps`](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops) instead.